server/base/src/main/java/org/apache/accumulo/server/compaction/FileCompactor.java

```diff
@@ -441,11 +438,7 @@ public CompactionStats call() throws IOException, CompactionCanceledException {
         iters.add(iter);
 
       } catch (Exception e) {
-
-        ProblemReports.getInstance(context).report(new ProblemReport(extent.tableId(),
-            ProblemType.FILE_READ, dataFile.getNormalizedPathStr(), e));
-
-        log.warn("Some problem opening data file {} {}", dataFile, e.getMessage(), e);
+        log.warn("Some problem opening data file {} {} {}", dataFile, extent, e.getMessage(), e);
```

Comment:
For these log messages, I don't think it's useful to print the `e.getMessage()` separately from the stack trace like this. Sometimes, it can be useful if it makes sense in context, but here, it's just printing it rather than providing any additional context. So, it's enough to just include the exception, which will automatically include the exception's message in the stack trace.

```suggestion
        log.warn("Some problem opening data file {} {}", dataFile, extent, e);
```

I think this same change can be made for all of these.